### DISCUSSION TITLE
Hammer editor

### DISCUSSION THREAD

**Discussion by 5588665q** [ValveSoftware/Dota2-Gameplay#10293] [General]
How i can create new tile setHow can I create a new tile set? Specifically, I want to use a different material instead of the standard ground, but it looks out of sync. How can I fix this? I have been looking for information about this for a long time and asked people to help with this, but no one can tell me about it. help me please, can u give me.

**Comment by bernadinerandall** [comment]
If you're experiencing issues with tiles appearing out of sync, ensure that you're aligning your new tiles correctly and checking settings such as tile size, pivot points, and positioning. Debugging tools in your engine can help you identify and resolve these issues.
[www.tellpopeyes.com](https://www.tellpopeyes.biz/)
